cloudformation/build-bundle-test-workflow.yml

```diff
@@ -0,0 +1,59 @@
+Description: Template for setting up resources for the build-bundle-test workflows
+Resources:
+  BuildBucket:
+    Type: "AWS::S3::Bucket"
+  BuildUser:
+    Type: "AWS::IAM::User"
```

Comment:
Why are we creating users?  Won't the scripts be run inside of the Jenkins system, or on a developer machine where they could assume a specific role.

Reply:
We would like to trigger the bundle builds & assembly fro Github Workflows which requires a User.

I don't think we need two sets of these creds though.  Perhaps we create a single user that is allowed to assume both these roles.

Reply:
Yeah, I'll switch to roles and create a single "system" user that can have credentials in the CI system.

Reply:
Done